src/app/lines.ts

```diff
@@ -2,35 +2,35 @@
  * Created by dafre on 5/11/2017.
  */
 
-import {Helpers} from "./helpers";
-import {LineVertex} from "./lineVertex";
-import {GlobeViewType, Settings} from "./settings";
-import {GlBuffer} from "./gl";
-import {ColorMap} from "./ColorMap";
+import {Helpers} from './helpers';
+import {LineVertex} from './lineVertex';
+import {GlobeViewType, Settings} from './settings';
+import {GlBuffer} from './gl';
+import {ColorMap} from './ColorMap';
 
 export class Lines {
 
-  // Variables
-  latLinesVertexPositionBuffer : any[];
-  latLinesVertexColorBuffer : any[];
-  lonLinesVertexPositionBuffer : any[];
-  lonLinesVertexColorBuffer : any[];
-  currDegreeStep : number;
-  LineVertex : LineVertex;
-
-  CoastsVertex : LineVertex;
-  LatLonVertex : LineVertex;
-  MinorIslandsVertex : LineVertex;
-  LakesVertex : LineVertex;
-  RiversVertex : LineVertex;
-  GeoLinesVertex : LineVertex;
-  TimeZonesVertex : LineVertex;
-
-  private _settings : Settings;
-  private _gl : WebGLRenderingContext;
-  private _colorMap : ColorMap;
-
-  public constructor(gl : WebGLRenderingContext, settings: Settings, colorMap : ColorMap) {
+  // letiables
```

Comment:
Looks to me like you did a blanket search for "var" and replaced them with "let", not a safe thing to do. It caused an issue here.